lib/auxiliary/time_helper.rb

```diff
@@ -4,9 +4,11 @@
 module AutoHCK
```

Comment:
`Signed-off-by: Vitalii Chulak <vchulakv@gmail.com>`: use @daynix mail there

`fix time to second helper`:  commit title starts with an uppercase letter
`time_to_second` is a function, so better to write its name as is in the commit message without any splitting